### ISSUE TITLE
Met instanceNormalizationPlugin.cu (292) - Cudnn Error in enqueue: 3 When try to convert ONNX to TRT in INT8

### ISSUE THREAD

**Issue by lannyyip** [NVIDIA/TensorRT#1610]
## Environment

**TensorRT Version**: 
**NVIDIA GPU**: RTX 3070
**NVIDIA Driver Version**:  470.63.01
**CUDA Version**: 11.3
**CUDNN Version**: 8.2.2.26
**Operating System**: Ubuntu 20.04
**Python Version (if applicable)**: 3.6
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.7
**Baremetal or Container (if so, version)**: Baremetal

## Relevant Files
[ONNX File](https://www.dropbox.com/s/00uspbi8ilr6gm4/u_net_model.onnx?dl=0)
[Calibration File](https://www.dropbox.com/s/e3y10cgjgawtvko/calibration.cache?dl=0)
[trtexec log](https://www.dropbox.com/s/pfwa2zvxepx4iu0/output.log?dl=0)

## Steps To Reproduce
1. Convert the trained model to ONNX format
2. Generate Calib file
3. Try to convert to TensorRT Engin in INT8, with the command below:
```shell
./trtexec --onnx=my_model.onnx  --optShapes=input0:1x1x1024x500 --int8 --calib=calibration.cache  --workspace=5000 --verbose
```
and get the core dumped as below:
```shell
[11/01/2021-22:35:33] [E] [TRT] instanceNormalizationPlugin.cu (292) - Cudnn Error in enqueue: 3 (CUDNN_STATUS_BAD_PARAM)
terminate called after throwing an instance of 'nvinfer1::plugin::CudnnError'
  what():  std::exception
Aborted (core dumped)
```
The detailed log please refer to he [log file](https://www.dropbox.com/s/pfwa2zvxepx4iu0/output.log?dl=0).

Similar core dumped is shown where I try to convert with python scripts
```shell
[TensorRT] ERROR: instanceNormalizationPlugin.cu (292) - Cudnn Error in enqueue: 3 (CUDNN_STATUS_BAD_PARAM)
terminate called after throwing an instance of 'nvinfer1::plugin::CudnnError'
  what():  std::exception
Aborted (core dumped)
```
I checked the code, it asserted in
```shell
        CUDNNASSERT(cudnnSetTensor4dDescriptor(mXDescriptor, CUDNN_TENSOR_NCHW, cudnn_dtype, 1, n * c, h, w));
```

It is OK to convert the model to FP16.
It is also OK to convert the model to INT8 if I change the InstanceNorm with BatchNorm.

Please help to investigate and have it fixed.

Lanny

**Comment by ttyio** [comment]
close since duplicate to https://github.com/NVIDIA/TensorRT/issues/1611, thanks!
